File: .ci/doc/templates/without-ctor.tpl.java
import io.kuzzle.sdk.Kuzzle;
import io.kuzzle.sdk.protocol.WebSocket;

public class SnippetTest {
  public static void main(String[] argv) {
    try {
      [snippet-code]
      System.out.println("Success");
    } catch (Exception e) {
      System.err.println((e.getMessage()));
    }
  }
}

File: .ci/doc/templates/without-ctor.tpl.kt
import io.kuzzle.sdk.Kuzzle
import io.kuzzle.sdk.protocol.WebSocket
import java.util.concurrent.ConcurrentHashMap
import java.util.concurrent.ExecutionException

fun main() {
  try {
    [snippet-code]
    println("Success")
  } catch (e: Exception) {
    e.printStackTrace()
  }
}
